HW03/HW3.md

```diff
@@ -0,0 +1,172 @@
+HW2
+================
+han
+2/4/2021
+
+## R Markdown: Exercise 1
+
+``` r
+#Exercise 1.1
+#Use filter() to subset the gapminder data to three countries of your choice in the 1970’s.
+
+Ex1.1=gapminder %>% 
+  filter(country == "Mexico"& year %in% 1970:1980|country =="India"& year %in% 1970:1980|country =="China" & year %in% 1970:1980)
+
+Ex1.1
+```
+
+    ## # A tibble: 6 x 6
+    ##   country continent  year lifeExp       pop gdpPercap
+    ##   <fct>   <fct>     <int>   <dbl>     <int>     <dbl>
+    ## 1 China   Asia       1972    63.1 862030000      677.
+    ## 2 China   Asia       1977    64.0 943455000      741.
+    ## 3 India   Asia       1972    50.7 567000000      724.
+    ## 4 India   Asia       1977    54.2 634000000      813.
+    ## 5 Mexico  Americas   1972    62.4  55984294     6809.
+    ## 6 Mexico  Americas   1977    65.0  63759976     7675.
+
+``` r
+#Exercise 1.2
+#Use the pipe operator %>% to select “country” and “gdpPercap” from your filtered dataset in 1.1.
+
+Ex1.1 %>% 
+  select(country,gdpPercap)
```

Comment:
I like how you wrote the code for this exercise in a simple way. It is easy to understand.